new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonTestUtils.java

```diff
@@ -448,6 +493,125 @@ public static HelmParams installAndVerifyVoyager(String voyagerNamespace,
     return voyagerHelmParams;
   }
 
+  /**
+   * Uninstall Elasticsearch.
+   *
+   * @return the Elasticsearch uninstallation parameters
+   */
+  public static boolean uninstallAndVerifyElasticsearch(LoggingExporterParams params) {
```

Comment:
could you please correct java doc description : add param, and correct return description ?

Reply:
fixed